api/views/application.py

```diff
@@ -16,6 +18,77 @@ def return_exception(e):
     }
     return jsonify(response_object)
 
+@application.route('/user/applications/upload', methods=['POST'])
+def upload(acl="public-read"):
+    resume_file = request.files['resume']
+    profile_pic_file = request.files['profile-pic']
+    video_file = request.files['elevator-pitch']
+    s3_client = boto3.client('s3')
+    success = 'files uploaded.'
```

Comment:
I think getting rid of this and just directly putting the string message in the payload is more clean!